gensim/test/test_scripts.py

```diff
@@ -105,9 +106,11 @@ def test_segment_and_write_all_articles(self):
         self.assertEqual(section_titles, self.expected_section_titles)
 
         # Check interlinks
-        self.assertTrue(interlinks['self-governance'] == 'self-governed')
-        self.assertTrue(interlinks['Hierarchy'] == 'hierarchical')
-        self.assertTrue(interlinks['Pierre-Joseph Proudhon'] == 'Proudhon')
+        # NOTE: after loading the interlinks will be loaded as list of lists instead of list of tuples
```

Comment:
What's the point of this note?

Reply:
This is there to ensure while writing tests after loading the json file, one does not try to compare against a tuple, as json always stores things as lists.

Reply:
I think it'd be more helpful if you include your response above in the message.

The current note kind of looks like it's stating the obvious, and doesn't add any value to the code.

Reply:
Just to clarify, are you suggesting that I change the comment to this: 

```python
# NOTE: To ensure while writing tests which load the generated json file, we need to ensure that the comparison of interlinks value is not done against a tuple, as json always stores things as lists. After loading, the interlinks will be loaded as list of lists instead of list of tuples. 
```

Reply:
Ugh, that's a bit too wordy and indirect. How about something shorter and more direct?

```python
# JSON has no tuples, only lists. So, we convert lists to tuples explicitly before comparison.
self.assertEqual(tuple(blah_list), foo_tuple)
```

Reply:
I like this much better.

Reply:
@mpenkov done.